build-tests/x86/leap/test-image-disk-selinux/appliance.kiwi

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="7.5" name="kiwi-test-image-disk-selinux" displayname="Selinux">
+    <description type="system">
+        <author>Marcus Schäfer</author>
+        <contact>marcus.schaefer@gmail.com</contact>
+        <specification>Simple selinux disk, devel policy</specification>
+    </description>
+    <preferences>
+        <version>1.15.4</version>
+        <packagemanager>zypper</packagemanager>
+        <locale>en_US</locale>
+        <keytable>us</keytable>
+        <timezone>Europe/Berlin</timezone>
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <rpm-check-signatures>false</rpm-check-signatures>
+        <bootsplash-theme>bgrt</bootsplash-theme>
+        <bootloader-theme>openSUSE</bootloader-theme>
+        <type image="oem" filesystem="xfs" kernelcmdline="console=ttyS0 security=selinux selinux=1 enforcing=0" firmware="uefi" format="qcow2">
```

Comment:
Is this any good if we're not testing enforcing mode?

Reply:
good point, I change it

Reply:
I blindly took over the recommendations from the openSUSE documentation